packages/edit-site/src/components/editor/index.js

```diff
@@ -126,7 +126,10 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 			hasSiteIcon: !! siteData?.site_icon_url,
 		};
 	}, [] );
-	useEditorTitle();
+	useEditorTitle(
+		postWithTemplate ? contextPostType : editedPostType,
+		postWithTemplate ? contextPostId : editedPostId
```

Comment:
Is this actually working? `postWithTemplate` is only assigned after this call.

Reply:
good call.